app/src/main/AndroidManifest.xml

```diff
@@ -20,9 +26,21 @@
             </intent-filter>
         </activity>
         <activity
-            android:name=".MainActivity"
+            android:name=".GVLExampleActivity"
             android:label=""
-            android:screenOrientation="portrait" />
+            android:screenOrientation="portrait">
+            <intent-filter
+                android:label="@string/app_name"
+                tools:ignore="AppLinkUrlError">
+                <action android:name="android.intent.action.VIEW" />
+                <action android:name="android.intent.action.SEND" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+
+                <data android:mimeType="image/*" />
```

Comment:
I think you should specify only the supported formats here, since for instance _bmp_ is not supported.